src/components/container/SurveyDataChart/SurveyDataChart.stories.tsx

```diff
@@ -0,0 +1,545 @@
+import React from "react";
+import { Card, Layout } from "../../presentational";
+import SurveyDataChart, { SurveyDataChartProps } from "./SurveyDataChart";
+import { SurveyAnswersPage } from "@careevolution/mydatahelps-js";
+
+export default { title: "Container/SurveyDataChart", component: SurveyDataChart, parameters: { layout: 'fullscreen' } };
+let render = (args: SurveyDataChartProps) => <Layout colorScheme="auto"><Card><SurveyDataChart {...args} /></Card></Layout>
+
+export const ffwelCreativeSelfLineChart = {
+    args: {
+        title: "FFWEL",
```

Comment:
nit on the title. I can see reading the code where it came from, but without that context just looking at storybook it looks like a typo